src/tools/dartaotruntime.md

```diff
@@ -0,0 +1,23 @@
+---
+title: dartaotruntime
+description: Command-line tool for running AOT-compiled snapshots of Dart code.
+toc: false
+---
+
+Use the `dartaotruntime` command to run AOT (ahead-of-time) compiled programs.
+This tool is supported on Windows, macOS, and Linux.
+Here's an example of using `dartaotruntime`:
+
+```terminal
+$ dartaotruntime main.aot
+Hello from Dart.
+```
+
+For information on command-line options, use the --help flag:
+
+```terminal
+$ dartaotruntime --help
+```
+
+For more information, see the
```

Comment:
I would move this up right after the example, and make it a bit more explicit, e.g. `To compile Dart programs into AUT snapshots compatible with this tool, see the ...`